### ISSUE TITLE
Badges' Markdown is messed up

### ISSUE THREAD

**Issue by Annihil** [FullHuman/tosin#1]
**Describe the bug**
Badges' markdown in the readme is a mess

**To Reproduce**
1. Check the badges of the README.md on GitHub
2. Notice the utter mess
3. Start raging

**Expected behavior**
Badges

**Result**
Garbage

**Screenshots**
![](https://i.imgur.com/h6TMmGh.png)

**Desktop (please complete the following information):**
 - OS: Mac OS X
 - Latest

**Comment by contrarian** [comment]
The badges are fucked? That's totally unacceptable! It's like a fucking badge of incompetency! 

I trust good sir that you will take corrective action promptly. 

Cheers,
   The Biggest G

**Comment by alinaish** [comment]
Hi good sirs @Annihil @contrarian

I think we all should learn from this great man @Ffloriel  how to write our packages. I use this package for my new blockchain bleeding edge project "Hello world" and it works perfectly.

Yes, there are some issues with markdown but have you seen package.json? It's a new level of software development. 

Relax,
🚫 🚫 🚫

**Comment by Ffloriel** [comment]
Hi @Annihil @contrarian @alinaish ,

Thank you for creating this issue and letting me know about this problem.
@Annihil , I appreciate you took the time to write the reproducing steps, it helped.
@contrarian I took action as soon as I saw your comment, it should be partially fixed now.

@alinaish, it is perfectly adapted to blockchain bleeding edge project "Hello world". I'm happy to know it's working for you. You will see it works great with new generation artificial intelligence package "Hello world" as well.

Stay awesome,
Oyster Black
